vercel_runtime/src/response.rs

```diff
@@ -81,7 +81,7 @@ where
 ///   Response::new(Body::from("hello")).body()
 /// );
 /// ```
-pub trait IntoResponse {
```

Comment:
Any reason we need all the `impl` for the trait still?

Reply:
no, let's remove it.